apps/pie-docs/src/accessibility/accessibility-overview.md

```diff
@@ -0,0 +1,88 @@
+
+Just as a good recipe caters to the diverse tastes and dietary needs of all diners, a design system that incorporates accessibility ensures that all users, **regardless of their abilities and disabilities**, can access and use digital products and services.
+
+Incorporating accessibility into a design system can lead to a **better user experience**, **increased engagement**, and **increased customer loyalty**.
+
+Finally, it’s important to highlight that **designing with accessibility in mind can help organisations comply with accessibility standards and regulations**, avoiding legal and financial consequences.
+
+---
+
+## How we build inclusive experiences
+
+Using our components is a great way to improve accessibility and consistency when building products for JustEatTakeaway.com.
+
+The component library in this documentation site includes code that we can use across applications, and this code includes **accessible markup**. Since the code exists in a single component that gets reused, it's easier to update and fix any bugs.
+
+The build-it-once, use-it-everywhere model means the **accessibility knowledge** of the designers and developers who build these components is available to all of JET and our partners.
+
+Many accessibility features come free in the components, but it's important to make sure they are integrated correctly, **in a fully accessible way**. Depending on how components are used, there may be more design and implementation considerations.
+
+
+---
+
+## Assistive technology support
+
+Our components are tested for accessibility with automated and manual techniques. Users should expect to be able to access features built with our components using **modern assistive technologies**, including native and **third-party tools** like screen readers, speech recognition programs, supports for low vision and colour blindness, alternative keyboards, switch devices, and tools for readability.
+
+{% notification {
+  type: "information",
+  message: "Find out more about automated and manual techniques in our [testing for accessibility](/accessibility/testing-for-accessibility) page."
+} %}
+
+---
+
+## Meeting global accessibility guidelines
+
+PIE is built to meet the Web Content **Accessibility Guidelines (WCAG)** 2.1 Level A and Level AA success criteria.
+
+In addition to meeting the previously mentioned accessibility guidelines, we also strive to provide a highly usable experience for everyone. Our goal is to make it easy for everyone to use our components, whether they have a disability or not.
+
+To learn more about the guidelines we adhere to, check out these resources:
+
+{% card {
+  items: [
+        {
+          linkText: "Inclusion, Diversity & Belonging at JET",
+          href: "https://kitchen.just-eat.com/home/inclusion-diversity-belonging-at-jet-update",
+          src: "../../../assets/img/accessibility/accessibility-overview/jet-guidelines.svg",
+          shouldOpenInNewTab: true
```

Comment:
Why is this the only one to open in a new tab?

Reply:
Anecdotally, not sure if this is the intended purpose but when I see this icon I expect a new tab/window to open :D 

<img width="44" alt="Screenshot 2023-05-16 at 09 06 08" src="https://github.com/justeattakeaway/pie/assets/16766185/985a82bf-3430-4bf4-9e49-2c7008f6b654">

Reply:
Because it links to the JET kitchen and you have to use Okta to access it